src/target/riscv32.c

```diff
@@ -153,6 +153,9 @@ static ssize_t riscv32_reg_read(target_s *target, const uint32_t reg, void *data
 		return riscv32_bool_to_4(riscv_csr_read(hart, RV_DPC, data));
 	if (reg >= RV_CSR_GDB_OFFSET)
 		return riscv32_bool_to_4(riscv_csr_read(hart, reg - RV_CSR_GDB_OFFSET, data));
+	if(reg >= RV_FPU_GDB_FIRST_REGISTER) {
```

Comment:
Please drop the additional braces in this and the register write if statement additions - they're not necessary. To be more consistent with the style of the existing code and how things are done in the other architecture implementations, may we suggest `RV_FPU_GDB_OFFSET` be used for this constant?

Please also run clang-format - there's an extra space in the second change here, and this + the other added if statement are missing one between `if` and the `(` for the condition.